Samples/office-add-in-commands/auto-open-task-pane/home.html

```diff
@@ -0,0 +1,33 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <title>Word Add-In with Commands Sample</title>
```

Comment:
Should this sample have a more descriptive title?

Reply:
changed.